ignite/contrib/metrics/average_precision.py

```diff
@@ -37,5 +41,7 @@ def activated_output_transform(output):
 
     """
 
-    def __init__(self, output_transform=lambda x: x):
-        super(AveragePrecision, self).__init__(average_precision_compute_fn, output_transform=output_transform)
+    def __init__(self, output_transform=lambda x: x, check_compute_fn=False):
```

Comment:
Typehint is missing on the new boolean parameter.

Reply:
Added!